app-creator/client/src/widgets/tool-bar/tool-bar.ts

```diff
@@ -283,12 +408,21 @@ export class ToolBar extends LitElement {
       importTemplate,
       clearTextArea,
       handleDuplicateButtonAction,
+      handleSnippetToggle,
       copy,
     } = this;
 
     const exportDialog = html`
       <paper-dialog id="export-dialog" with-backdrop no-cancel-on-outside-click>
-        <h2>Paste string in EE Code Editor</h2>
+        <h2>Paste string in the Earth Engine Code Editor</h2>
+        <paper-tabs selected=${this.selectedTab} noink>
+          <paper-tab @click=${() => handleSnippetToggle.call(this, 0)}>
+            Code
+          </paper-tab>
+          <paper-tab @click=${() => handleSnippetToggle.call(this, 1)}>
+            JSON
+          </paper-tab>
```

Comment:
You should be able to iterate on the key/values of the proposed enum above to produce these `paper-tab` elements dynamically.